### ISSUE TITLE
Estimated Impact  Confusion

### ISSUE THREAD

**Issue by mgifford** [mgifford/daily-dap#106]
I am confused by this:

Estimated Impact (daily)[#](https://mgifford.github.io/daily-dap/docs/reports/daily/2026-03-20/index.html#estimated-impact-heading)
No accessibility findings data available for this scan. The impact estimation requires detailed accessibility findings from scanning tools. Currently, the scanner is running in a mode that does not collect individual accessibility issues.

Americans Excluded by Disability[#](https://mgifford.github.io/daily-dap/docs/reports/daily/2026-03-20/index.html#fpc-exclusion-heading)
No accessibility findings data is available to estimate excluded users by disability category for this scan.

I don't know where this would be or what might be missing. Is there a GitHub Action that needs to be triggered to generate the data that is missing?  Actually, why isn't the No accessibility findings data available?

Isn't this all available here:
https://mgifford.github.io/daily-dap/docs/reports/daily/2026-03-20/index.html#axe-patterns-heading

What is this report waiting on? 

https://github.com/mgifford/daily-dap/blob/762e213ec18603229766782f37a474b7543b3b7c/src/publish/render-pages.js#L987
  if (!hasImpactData && affectedSharePercent === 0) {

And 
https://github.com/mgifford/daily-dap/blob/762e213ec18603229766782f37a474b7543b3b7c/src/publish/render-pages.js#L1017
  if (categories.length === 0) {

Why is this the case?
